Packs/ForcepointDLP/Integrations/ForcepointEventCollector/README.md

```diff
@@ -1,17 +1,24 @@
-Use this integration to fetch security incidents from Forcepoint DLP as Cortex XSIAM events.
+Forcepoint Data Loss Prevention (DLP) enables businesses to discover, classify, monitor, and protect data intuitively with zero friction to the user experience. Audit behavior in real-time with Risk-Adaptive Protection to stop data loss before it occurs.
+This integration was integrated and tested with version 10.2 of ForcePointDLP.
 
-## Configure Forcepoint DLP Event Collector in Cortex
+## Configure ForcePoint DLP on Cortex XSOAR
 
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for ForcePoint DLP.
+3. Click **Add instance** to create and configure a new integration instance.
 
-| **Parameter**                      | **Required** |
-|------------------------------------|--------------|
-| Server URL                         | True         |
-| API Key                            | True         |
```

Comment:
Dont we use the API key anymore?

Reply:
Split to two integrations, this now exists only within the collector